TARSbot/commands/Commands.cs

```diff
@@ -325,6 +326,12 @@ public static void Init(Dictionary<string, Func<CommandArgs, Task>> commands)
         {
             await e.Channel.SendMessage(String.Format("My invite link is:\nhttps://discordapp.com/oauth2/authorize?&client_id={0}&scope=bot&permissions=0", ConstData.clientId));
         }
+        
+        public static async Task Github(CommandArgs e)
+        {
+            await e.Message.Delete();
+            await e.Channel.SendMessage(@"<@106272491042787328> was trying to mess with me!");
```

Comment:
HERE IT IS. bloody mentions, going arount in @name#number form... what were they thinking!?